src/main/resources/data.sql

```diff
@@ -26,18 +23,16 @@ insert into phase (kind, description) values ('RESOURCE_ALLOCATION', 'Cloud Reso
 insert into phase (kind, description) values ('CHECKPOINT_MEETING', 'Progress meeting at halfway point finished');
 insert into phase (kind, description) values ('CODE_REVIEW', 'Code Review of work completed in Iteration has commenced');
 insert into phase (kind, description) values ('COMPLETE', 'Iteration has completed, ready to merge into upstream');
-
 -- insert tags
 insert into tag (name, description) values ('Angular', 'This project used an Angular Frontend');
 insert into tag (name, description) values ('React', 'This project used a React Frontend');
 insert into tag (name, description) values ('JDBC', 'Uses JDBC to connect to the DB');
-insert into tag (name, description) values ('AWS RDS', 'This project used a React Frontend');
-insert into tag (name, description) values ('REST', 'This project used a React Frontend');
-insert into tag (name, description) values ('SpringMVC', 'This project used a React Frontend');
-insert into tag (name, description) values ('SPRING ORM', 'This project used a React Frontend');
-insert into tag (name, description) values ('GIT', 'This project used a React Frontend');
-
+insert into tag (name, description) values ('AWS RDS', 'This project used AWS RDS');
+insert into tag (name, description) values ('REST', 'This project used REST');
+insert into tag (name, description) values ('Spring MVC', 'This project used Spring MVC');
+insert into tag (name, description) values ('Spring ORM', 'This project used Spring ORM');
+insert into tag (name, description) values ('GIT', 'This project used GIT');
 -- insert projects
 insert into project(description, name, owner, status, phase) values('rideforce project', 'rideforce', 3, 3, 2);
-
-
+insert into project(description, name, owner, status, phase) values('Timmies project', 'Timmy', 1, 2, 3);
+insert into project(description, name, owner, status, phase) values('Running for mayor', 'Operation Mayor', 4, 3, 2);
```

Comment:
@Christian-Downs could you change the names of these to appear clearer and more professional?  (for example: "Caliber Staging Module", "Performance Managment" instead of Timmy and Operation Mayor?